src/bugle_forms/views/layout.clj

```diff
@@ -0,0 +1,24 @@
+(ns bugle-forms.views.layout
+  (:require [hiccup.page :refer [include-css]]))
+
+(defn- navbar [nav-items]
+  [:div {:class "nav"}
+   [:a {:href "/" :title "home"} "🎺"]
+   (map (fn [[link-title link-url]]
+          [:a {:href link-url :title link-title} link-title]) nav-items)])
```

Comment:
suggestion: Aligning function args (for `map` in this case) vertically makes it easier to read.
(https://guide.clojure.style/#vertically-align-fn-args)